CharacterCreation/04_03_SMGs_&_Shotguns.txt

```diff
@@ -283,36 +382,53 @@ Auto Fire Rate: 	4
 Armor Piercing Level:	1	
 Reflex Modifier:	0	
 Suppression Level:	Medium	
-Notes/Description:	CQB weapon.
-Notes/Description: Slug shells do 110 damage for all range brackets.
-	Mag-fed, 2 Actions to reload.
+
+	CQB weapon.	Mag-fed, 2 Actions to reload.
 		
-----Frontier Plas-12	$3253	----
-Plasma Damage, Ranged Miss Chance:		
-	160 damage with a miss chance of 1 out to 10m	
-	140 damage with a miss chance of 3 out to 20m
-	110 damage with a miss chance of 4 out to 35m	
+== Frontier Plas-12	$3253	==
+
++---------+--------------+----------+
+| Damage  |  Miss Chance |  Out to  |
++---------+--------------+----------+
+|   160   |      1       |    10m   |   
++---------+--------------+----------+
+|   140   |      3       |    20m   |
++---------+--------------+----------+
+|   110   |4 (Slug only) |    35m   |

```

Comment:
This gun makes no mention of slugs in the deleted lines. Like one of the plasmas above, I suspect it was intended to be buck only.